README.md

```diff
@@ -49,7 +49,7 @@ With the preview release of PowerShellGet for PowerShell V3/V4, I am deprecating
 
 To install the build from GitHub, you can install using [PsGet](http://psget.net) (very easy to install), and run `Install-Module PSReadLine`.  Note that PsGet and PowerShellGet both have a Install-Module command, but they are very different despite having similar names and commands. PowerShellGet is implemented and supported by the PowerShell team at Microsoft, PsGet is a 3rd party tool that was a source of inspiration for PowerShellGet.
 
-If you'd rather not use PsGet, you can just download the file [PSReadLine.zip](https://github.com/lzybkr/PSReadLine/releases/download/Latest/PSReadLine.zip) and extract the contents into your `C:\Users\[User]\Documents\WindowsPowerShell\modules\PSReadLine` folder. (You may have to create these directories if they don't exist.)
+If you'd rather not use PsGet, you can just download the file [PSReadLine.zip](https://github.com/PowerShell/PSReadLine/releases/download/Latest/PSReadLine.zip) and extract the contents into your `C:\Users\[User]\Documents\WindowsPowerShell\modules\PSReadLine` folder. (You may have to create these directories if they don't exist.)
```

Comment:
Since PowerShellGet was released, I haven't been updating the "Latest" release (which depends on a git tag name `latest`) that `PSGet` relies on - so that just installs 1.2 right now.  Maybe we should just remove any mention of `PSGet`.

Reply:
That sounds good. And also I think we can remove mentioning 'download the PSReadLine.zip' since we are not going to publish on GitHub.